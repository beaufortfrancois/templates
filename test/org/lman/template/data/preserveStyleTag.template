<html>
<style>
@media only screen and (max-width: 580px){.button{margin:20px 0}}
{{name}}
</style>
<style>@media only screen and (max-width: 580px){.button{margin:20px 0}}</style>
<h1>My name is {{name}}</h1>
<style>@media only screen and (max-width: 580px){.button{margin:20px 0}}</style>
</html>
