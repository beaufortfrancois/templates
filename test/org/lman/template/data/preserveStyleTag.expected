<html>
<style>
@media only screen and (max-width: 580px){.button{margin:20px 0}}
</style>
<style>@media only screen and (max-width: 580px){.button{margin:20px 0}}</style>
<h1>Party shopping list</h1>
<ul>
  <li>Ben - Spaghetti</li>
</ul>
<style>@media only screen and (max-width: 580px){.button{margin:20px 0}}</style>
</html>
